Changelog.md

```diff
@@ -3,6 +3,7 @@
 Language Features:
  * Allow to obtain the selector of public or external library functions via a member ``.selector``.
  * Parser: Allow splitting string and hexadecimal string literals into multiple parts.
+ * Inline Assembly: Support referencing other constants in inline assembly.
```

Comment:
Nit: `in inline assembly` could be removed since the changelog entry is under the `Inline Assembly` heading

Reply:
Fixed it here https://github.com/ethereum/solidity/pull/7874/commits/b4f815c8718d3ff1f528cb242af7bed044e4ca8f